lib/messaging/controls/handler.rb

```diff
@@ -42,6 +42,47 @@ class Example
           end
         end
       end
+
+      module SessionArgument
+        class Example
+          include Messaging::Handle
+
+          attr_accessor :session
+
+          def configure(session: nil)
+            self.session = session
+          end
+        end
+
+        module Invalid
```

Comment:
I think there's an established pattern of referring to these things as "Anomaly", rather than "Invalid". There's a remnant in Identifier::UUID, I believe, but that's because that library hasn't been brought up to date (still uses test/bench, etc).

Reply:
Thanks, it's corrected.